package.json

```diff
@@ -337,7 +333,7 @@
     "react-textfit": "1.1.0",
     "rebass": "3.0.0-9",
     "redux": "4.0.1",
-    "redux-electron-ipc": "1.2.0",
+    "redux-electron-ipc": "LN-Zap/redux-electron-ipc#b513220d085ad3e96e459d7efcdfc37bf75417b6",
```

Comment:
Because `redux-electron-ipc` tries to get hold of the `ipcRenderer` by doing `require('electron')` but when `nodeIntegration` is disabled, `electron` can not be used in the renderer process.

I have made `ipcRenderer` available at `window.ipcRenderer` in the `preload.js` and I forked `redux-electron-ipc` so that it uses that reference to `window.ipcRenderer` instead of trying to `require('electron')`.

See https://github.com/mariotacke/redux-electron-ipc/commit/a1c08ce9f5897a57eb0473afa76a41b39b1e0712